CommunityCenterDB/4 data.sql

```diff
@@ -0,0 +1,26 @@
+use CommunityCenterDB
+go
+delete ExpenseLedger
+go
+insert ExpenseLedger(VendorName, VendorAddress, Cause, SubCause, PaymentAmount, PaymentDate, SatisfactionRate)
+select 
+    case when m.medal = 'gold' then concat('www.', m.FirstName, m.LastName, '.com')
+         when m.medal = 'silver' then concat('www.', m.FirstName, m.LastName, '.net')
+         when m.medal = 'bronze' and m.Season = 'summer' then concat(m.FirstName, ' ', m.LastName, ' & friends')
+         when m.medal = 'bronze' and m.Season = 'winter' then concat(m.FirstName, ' ', m.LastName, ' company')
+    end,
+    m.OlympicLocation,
+    m.Sport,
+    concat(substring(m.Sport, 1, 5), ' - ', m.SportSubcategory),
+    case when m.AgeofMedalist <= 20 then m.YearBorn * m.AgeofMedalist
+         when m.AgeofMedalist >= 35 then m.YearBorn + m.OlympicYear
+         else m.OlympicYear / m.AgeofMedalist
+    end,
+    datetimefromparts(m.OlympicYear, 01, len(m.LastName), m.AgeofMedalist/8, len(m.Sport), 52, len(m.FirstName + m.LastName + m.OlympicLocation) * 10),
+--It makes more sense for table to have an int column and the desc computed off of that. However, for this data then I had to do the opposite to get the correct results.
```

Comment:
This way is nicer for a real life scenario, because than any time you would want to add data to your table (not based off of president, just inserting sample data) all you have to do is insert an int and the rest is computed. Here this is one report that it had to be done the opposite but for the future the way I had done it is much more helpful. Understand?

Reply:
Yes. Thank you for clarifying